docs/en/query_language/dicts/external_dicts.md

```diff
@@ -31,6 +32,8 @@ The dictionary configuration file has the following format:
 
 You can [configure](external_dicts_dict.md) any number of dictionaries in the same file.
 
+Dictionaries [DDL queries](../create.md#create-dictionary-query) doesn't require any additional records in servers' xml-config. They allows to work with dictionaries as first-class entities, like tables or views.
```

Comment:
```suggestion
[DDL queries for dictionaries](../create.md#create-dictionary-query) doesn't require any additional records in server configuration. They allow to work with dictionaries as first-class entities, like tables or views.
```

Also at some point we could rewrite this so DDL sounds like a recommended way, and xml is legacy/deprecated.

Reply:
Ok, but not now.